roboteam_ai/src/skills/Attack.cpp

```diff
@@ -15,24 +15,63 @@ void Attack::onInitialize() {
     ownGoal = properties->getBool("ownGoal");
```

Comment:
This whole file feels very hacky with many hardcoded constants now. Why not simply do something like checking for all geneva states which angle would correspond to a nice angle towards the goal, and then picking the best one? 
In general it doesn't feel very 'procedurial'; it would help for exmaple if it has more the structure of something that has 2 modules, 1 module that gets a target and subsequently decides at what geneva angle to shoot and another module that executes the decided behavior.  I feel like there are many more intuitive and reliable ways to use the geneva.